packages/mui-material/src/Typography/Typography.js

```diff
@@ -80,23 +113,28 @@ const defaultVariantMapping = {
   inherit: 'p',
 };
 
-// TODO v6: deprecate these color values in v5.x and remove the transformation in v6
+// TODO v7: remove this transformation and `extendSxProp`
 const colorTransformations = {
-  primary: 'primary.main',
   textPrimary: 'text.primary',
-  secondary: 'secondary.main',
   textSecondary: 'text.secondary',
-  error: 'error.main',
-};
-
-const transformDeprecatedColors = (color) => {
-  return colorTransformations[color] || color;
+  // For main palette, the color will be applied by the styles above.
```

Comment:
Here, "styles above" refers to lines 65-72, right?

Maybe add that clarification to the comment, as "styles above" might be understood as the lines directly above.

Reply:
Referencing lines would probably not work though, so I would reference it like this:

```suggestion
  // For the main palette, the color will be applied by the ...Object.entries(theme.palette) clause in the TypographyRoot's styles
```